lib/govuk_publishing_components/presenters/machine_readable/organisation_schema.rb

```diff
@@ -18,11 +18,77 @@ def structured_data
           },
           "name" => page.title,
           "description" => page.description || page.body,
-        }.merge(parent_organisations).merge(sub_organisations).merge(search_action)
+        }
+        .merge(members)
+        .merge(parent_organisations)
+        .merge(sub_organisations)
+        .merge(search_action)
       end
 
     private
 
+      def members
+        return {} unless ministers.present?
+
+        members = ministers_with_grouped_roles.map do |person, roles|
+          person.merge("hasOccupation" => roles)
+        end
+
+        {
+          "member" => members
+        }
+      end
+
+      def ministers
+        page.content_item.dig("details", "ordered_ministers") || []
+      end
+
+      def ministers_with_grouped_roles
+        @ministers_with_grouped_roles =
```

Comment:
Does this need to be assigned to an instance variable? `@ministers_with_grouped_roles` is not used again or memoized

Reply:
Good point, I think I'd chosen the worst of both options :-)